# departamentos/domain/exception/departamento_exception.py

class DepartamentoError(Exception):
    pass
